src/ripple/app/tx/impl/OfferStream.h

```diff
@@ -166,17 +170,30 @@ class FlowOfferStream : public TOfferStreamBase<TIn, TOut>
 {
 private:
     boost::container::flat_set<uint256> toRemove_;
+    boost::container::flat_set<uint256> becameUnfunded_;
 protected:
     void
     permRmOffer (std::shared_ptr<SLE> const& sle) override;
 
+    void
+    becameUnfundedOffer (std::shared_ptr<SLE> const& sle) override;
+
 public:
     using TOfferStreamBase<TIn, TOut>::TOfferStreamBase;
 
-    boost::container::flat_set<uint256> const& toRemove () const
+    boost::container::flat_set<uint256> const& permUnfunded () const
     {
         return toRemove_;
```

Comment:
I'm wondering if `permUnfunded_` might be a better name than `toRemove_` for this container?